homeassistant/components/climate/generic_thermostat.py

```diff
@@ -57,6 +57,8 @@
     vol.Optional(CONF_TARGET_TEMP): vol.Coerce(float),
     vol.Optional(CONF_KEEP_ALIVE): vol.All(
         cv.time_period, cv.positive_timedelta),
+    vol.Optional(CONF_INITIAL_OPERATION_MODE):
```

Comment:
Set a default value.
default=STATE_AUTO

Reply:
The state is already auto if initial value is nulla.  If I set a default value I can't understand if I have to recover value from old state.

Reply:
:+1: